src/main/webapp/app/core/course/manage/quick-actions/quick-actions.component.html

```diff
@@ -0,0 +1,78 @@
+<div class="card col-5 mb-3">
+    <h2 class="card-header course-quick-actions-title mb-3" jhiTranslate="entity.quickActions"></h2>
+    <div class="card-body">
+        <div class="d-flex flex-wrap align-items-center mb-3">
+            <div class="d-flex flex-nowrap">
+                <div class="text-center me-4">
+                    <div class="small text-muted" jhiTranslate="artemisApp.course.students"></div>
+                    @if (course()?.isAtLeastInstructor) {
+                        <a [routerLink]="['/course-management', course()?.id, 'groups', 'students']" class="fw-bold" id="number-of-students">{{
+                            course()?.numberOfStudents || 0
+                        }}</a>
+                    } @else {
+                        <div class="fw-bold" id="number-of-students-text">{{ course()?.numberOfStudents || 0 }}</div>
+                    }
+                </div>
+                <div class="text-center me-4">
+                    <div class="small text-muted" jhiTranslate="artemisApp.course.tutors"></div>
+                    @if (course()?.isAtLeastInstructor) {
+                        <a [routerLink]="['/course-management', course()?.id, 'groups', 'tutors']" class="fw-bold" id="number-of-tutors">{{
+                            course()?.numberOfTeachingAssistants || 0
+                        }}</a>
+                    } @else {
+                        <div class="fw-bold" id="number-of-tutors-text">{{ course()?.numberOfTeachingAssistants || 0 }}</div>
+                    }
+                </div>
+                <div class="text-center me-4">
+                    <div class="small text-muted" jhiTranslate="artemisApp.course.editors"></div>
+                    @if (course()?.isAtLeastInstructor) {
+                        <a [routerLink]="['/course-management', course()?.id, 'groups', 'editors']" class="fw-bold" id="number-of-editors">{{ course()?.numberOfEditors || 0 }}</a>
+                    } @else {
+                        <div class="fw-bold" id="number-of-editors-text">{{ course()?.numberOfEditors || 0 }}</div>
+                    }
+                </div>
+                <div class="text-center me-2">
+                    <div class="small text-muted" jhiTranslate="artemisApp.course.instructors"></div>
+                    @if (course()?.isAtLeastInstructor) {
+                        <a [routerLink]="['/course-management', course()?.id, 'groups', 'instructors']" class="fw-bold" id="number-of-instructors">{{
+                            course()?.numberOfInstructors || 0
+                        }}</a>
+                    } @else {
+                        <div class="fw-bold" id="number-of-instructors-text">{{ course()?.numberOfInstructors || 0 }}</div>
+                    }
+                </div>
```

Comment:
Please define an ng-template for this instead of duplicating the code.

Also - why do all elements have `me-4` but instructors `me-2`?